common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -991,76 +991,75 @@ def test_save_problem_submitted_no_randomize(self, rerandomize):
 
     def test_check_button_name(self):
 
-        # If last attempt, button name changes to "Final Check"
         # Just in case, we also check what happens if we have
         # more attempts than allowed.
         attempts = random.randint(1, 10)
         module = CapaFactory.create(attempts=attempts - 1, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Final Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
         module = CapaFactory.create(attempts=attempts, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Final Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
         module = CapaFactory.create(attempts=attempts + 1, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Final Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
         # Otherwise, button name is "Check"
         module = CapaFactory.create(attempts=attempts - 2, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
         module = CapaFactory.create(attempts=attempts - 3, max_attempts=attempts)
-        self.assertEqual(module.check_button_name(), "Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
-        # If no limit on attempts, then always show "Check"
+        # If no limit on attempts, then always show "Submit"
         module = CapaFactory.create(attempts=attempts - 3)
-        self.assertEqual(module.check_button_name(), "Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
         module = CapaFactory.create(attempts=0)
-        self.assertEqual(module.check_button_name(), "Check")
+        self.assertEqual(module.check_button_name(), "Submit")
 
     def test_check_button_checking_name(self):
         module = CapaFactory.create(attempts=1, max_attempts=10)
-        self.assertEqual(module.check_button_checking_name(), "Checking...")
+        self.assertEqual(module.check_button_checking_name(), "Submitting...")
 
         module = CapaFactory.create(attempts=10, max_attempts=10)
-        self.assertEqual(module.check_button_checking_name(), "Checking...")
+        self.assertEqual(module.check_button_checking_name(), "Submitting...")
 
     def test_check_button_name_customization(self):
```

Comment:
This text customization is no longer working, so I would advocate just deleting the tests (all of the associated code will be deleted as part of https://openedx.atlassian.net/browse/TNL-4939). "custom_check" and "custom_final_check" are no longer used at all in capa_base.py, so all this test is actually doing now is ensuring what the hard-coded values of the button are.